src/components/map/Layers/Countries.tsx

```diff
@@ -45,31 +45,8 @@ function SetCountryEstimates(map: mapboxgl.Map, estimateGradePrevalences: Estima
     });
 }
 
-function SetMapData(map: mapboxgl.Map, estimateGradePrevalences: EstimateGradePrevalence[], countryFocus: any) {
-
+function SetMapData(map: mapboxgl.Map, estimateGradePrevalences: EstimateGradePrevalence[]) {
     SetCountryEstimates(map, estimateGradePrevalences);
-
-    if(countryFocus) // Shades following countries dark
```

Comment:
any reason why we are getting rid of this countryFocus behaviour? don't think it's essential but just want to be on the same page

Reply:
found a way to highlight the country without actually needing this mode

Reply:
We just limit the sero data shading to only one country, effecting making the countryfocus mode obsolete